swift/ql/lib/change-notes/2023-05-25-fix-ast-and-cfg-inconsistencies.md

```diff
@@ -0,0 +1,14 @@
+---
+category: fix
+---
+
+* Fixed some AST printing inconsistencies leading to a non-tree AST. In particular:
+    * `getOpaqueExpr()` is not considered a child of `OpenExistentialExpr` anymore, as it is
+       actually a reference to an expression nested within `getSubExpr()`;
+    * fixed some corner cases involving synthesized `PatternBindingDecl`s for variables wrapped with
+      property wrappers.
+* Fixed some control flow graph inconsistencies leading to multiple successors and dead ends.
+  In particular:
+    * fixed the corner cases mentioned above for AST printing, which were a problem also for the
+      control graph;
+    * fixed an inconsistency caused by an unneeded special treatment of `TapExpr`.
```

Comment:
I think this change note is too technical. I'd normally not even write a change note for these things. How about something like this instead?

```suggestion
---
category: fix
---

* Fixed a number of inconsistencies in the abstract syntax tree (AST) and in the control-flow graph (CFG). This may lead to more results in queries that use these libraries, or libraries that depend on them (such as dataflow).